modules/ROOT/pages/munit-cookbook.adoc

```diff
@@ -12,12 +12,12 @@ This section links to examples that demonstrate common MUnit usages.
 | xref:mock-before-after-foreach-cookbook.adoc[Mocking a Message Before and Inside a Foreach Processor] | Learn how to mock a processor that can return different payloads.
 | xref:test-mock-errors-cookbook.adoc[Testing and Mocking Errors] | Learn how to mock expected errors in your code.
 | xref:use-environment-properties-on-tests-cookbook.adoc[Testing with Environment Properties] | Learn how to load different environment values to your MUnit tests.
```

Comment:
Not a big deal, but do we need "Learn how to" everywhere?

Reply:
Hi! Not at all! I've made some changes. Thanks!